On darwin:
```sh
error: builder for '/nix/store/3y0hvrzhbx1d6y7yrzafds1zw0d02iib-python3.12-morecantile-5.4.2.drv' failed with exit code 1;
       last 10 log lines:
       > >       assert tms.bbox == tms_bbox
       > E       assert BoundingBox(l...6368, top=0.0) == (100.23646734...50376368, 0.0)
       > E
       > E         At index 0 diff: 100.23646734667147 != 100.23646734667152
       > E         Use -v to get more diff
       >
       > tests/test_morecantile.py:430: AssertionError
       > =========================== short test summary info ============================
       > FAILED tests/test_morecantile.py::test_tiles_when_tms_bounds_and_provided_bounds_cross_antimeridian[bounds0-6-32750-tms_bbox0] - assert BoundingBox(l...6368, top=0.0) == (100.23646734...50376368, 0.0)
       > =================== 1 failed, 226 passed, 3 xfailed in 4.45s ===================
```

--- a/pkgs/development/python-modules/morecantile/default.nix
+++ b/pkgs/development/python-modules/morecantile/default.nix
@@ -16,7 +16,7 @@
 
 buildPythonPackage rec {
   pname = "morecantile";
-  version = "5.3.0";
+  version = "5.4.2";
   pyproject = true;
   disabled = pythonOlder "3.8";
 
@@ -24,7 +24,7 @@ buildPythonPackage rec {
     owner = "developmentseed";
     repo = "morecantile";
     rev = version;
-    hash = "sha256-F7xYQrOngoRsZjmS6ZHRGN0/GD53AYcMQzyY1LZ1O7I=";
+    hash = "sha256-kUAde+6IUu95tFHFCB6kWoYsRf9GxR+gRJki/tvhIaY=";
   };
 
   nativeBuildInputs = [ flit ];

--- a/pkgs/development/python-modules/morecantile/default.nix
+++ b/pkgs/development/python-modules/morecantile/default.nix
@@ -1,5 +1,6 @@
 {
   lib,
+  stdenv,
   buildPythonPackage,
   fetchFromGitHub,
   pytestCheckHook,
@@ -40,6 +41,11 @@ buildPythonPackage rec {
     mercantile
     pytestCheckHook
     rasterio
+  ];
+
+  disabledTests = lib.optionals stdenv.isDarwin [
+    # https://github.com/developmentseed/morecantile/issues/156
+    "test_tiles_when_tms_bounds_and_provided_bounds_cross_antimeridian"
   ];
 
   pythonImportsCheck = [ "morecantile" ];

